files/image_config/hostcfgd/hostcfgd

```diff
@@ -219,6 +220,537 @@ class AaaCfg(object):
+            else:
+                cmd = "/sbin/dhclient {} -r {} && [ -f /var/run/dhclient{}.{}.pid ] && kill `cat /var/run/dhclient{}.{}.pid` && rm -f /var/run/dhclient{}.{}.pid".format(version, ifname, file_ext, ifname, file_ext, ifname, file_ext, ifname)
+
+            syslog.syslog(syslog.LOG_INFO, "Configure dhclient , cmd : {}".format(cmd))
+
+            subprocess.check_call(cmd, shell=True)
+        except subprocess.CalledProcessError as err:
+            syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}".format(err.cmd, err.returncode, err.output))
+            pass
+
+    def mgmt_intf_ip_prefix_set (self, ifname, ip_prefix, op):
+        try:
+            ver = "-4"
+            mgmt_conf = netaddr.IPNetwork(ip_prefix)
+            if mgmt_conf.version == 6:
+                ver = "-6"
+            cmd = "ip {} addr {} {} dev {}".format(ver, op.lower(), ip_prefix, ifname)
+            syslog.syslog(syslog.LOG_INFO, "Configure ip_prefix, op : {}, ip_prefix: {}, cmd : {}, mgmt_conf: {}".format(op, ip_prefix, cmd, mgmt_conf))
+            subprocess.check_call(cmd, shell=True)
+            cmd = "ip {} rule {} from {} table default pref 1003".format(ver, op.lower(), str(mgmt_conf.ip))
+            subprocess.check_call(cmd, shell=True)
+        except subprocess.CalledProcessError as err:
+            syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}".format(err.cmd, err.returncode, err.output))
+            pass
+
+    def handle_mgmt_port_cfg(self, key, data):
+        mgmt_intf =  self.app_db.get_entry(APP_MGMT_PORT_TABLE, key)
+        is_all = False
+        if mgmt_intf == {}:
+            is_all = True
+
+        try:
+            if (('mtu' in data) and ((is_all == True) or (mgmt_intf == {}) or ('mtu' not in mgmt_intf) or (data['mtu'] != mgmt_intf['mtu']))):
+                self.mgmt_intf_mtu_set(key, data['mtu'], mgmt_intf)
+
+            if (('description' in data) and ((is_all == True) or (mgmt_intf == {}) or ('description' not in mgmt_intf) or (data['description'] != mgmt_intf['description']))):
+                self.mgmt_intf_description_set(key, data['description'], mgmt_intf)
+
+            if (('admin_status' in data) and ((is_all == True) or (mgmt_intf == {}) or ('admin_status' not in mgmt_intf) or (data['admin_status'] != mgmt_intf['admin_status']))):
+                self.mgmt_intf_admin_status_set(key, data['admin_status'], mgmt_intf)
+
+            if (('autoneg' in data) and ((is_all == True) or (mgmt_intf == {}) or ('autoneg' not in mgmt_intf) or (data['autoneg'] != mgmt_intf['autoneg']))):
+                self.mgmt_intf_autoneg_set(key, data['autoneg'], mgmt_intf)
+
+            if (('speed' in data) and ((is_all == True) or (mgmt_intf == {}) or ('speed' not in mgmt_intf) or (data['speed'] != mgmt_intf['speed']))):
```

Comment:
Checks here seems to be cluttered. Please provide a comment for this section and also group it if possible?

Reply:
Addressed.